app/reducers/exercises.js

```diff
@@ -4,11 +4,13 @@ import axios from 'axios';
 
 export const RECEIVED_EXERCISES    = 'RECEIVED_EXERCISES'
 export const REMOVE_EXERCISES      = 'REMOVE_EXERCISES'
+export const REMOVE_ONE_EXERCISE   = 'REMOVE_ONE_EXERCISE'
 
```

Comment:
I think you left out CREATE_EXERCISE here and its associated action creator/reducer/dispatch below.